Frontend/src/Components/Battery/Battery.js

```diff
@@ -0,0 +1,143 @@
+import {Flex, Center, Box, Spacer, Text, useColorMode} from "@chakra-ui/react";
+import RangeRow from "../Shared/RangeRow";
+import RangeBar from "../Shared/RangeBar";
+import CONSTANTS from "../../data-constants.json";
+import getColor from "../Shared/colors";
+
+export default function Battery(props) {
+    const { colorMode } = useColorMode();
+    const headerBg = getColor("header", colorMode);
+    const borderColor = getColor("border", colorMode);
+
+    const fanEnableText = (props.data?.fan_enable[0] ?? -1.0) ? 'EN' : 'DIS';
+    const fanEnableColor = (props.data?.fan_enable[0] ?? -1.0) ? getColor("greenBg", colorMode) : getColor("redBg", colorMode);
+    const fanEnableTextColor = (props.data?.fan_enable[0] ?? -1.0) ? getColor("txtGreenBg", colorMode) : getColor("txtRedBg", colorMode);
+
+    const chargeEnableText = (props.data?.charge_enable[0] ?? -1.0) ? 'Enable' : 'Disable';
+    const chargeEnableColor = (props.data?.charge_enable[0] ?? -1.0) ? getColor("greenBg", colorMode) : getColor("redBg", colorMode);
+    const chargeEnableTextColor = (props.data?.charge_enable[0] ?? -1.0) ? getColor("txtGreenBg", colorMode) : getColor("txtRedBg", colorMode);
```

Comment:
```suggestion
    const chargeEnableText = ((props.data?.charge_enable[0] ?? -1.0) === -1.0) ? 'N/A' : (props.data.charge_enable[0] ? 'High' : 'Low');
    const chargeEnableColor = ((props.data?.charge_enable[0] ?? -1.0) == CONSTANTS.charge_enable.MAX) ? getColor("greenBg", colorMode) : getColor("redBg", colorMode);
    const chargeEnableTextColor = ((props.data?.charge_enable[0] ?? -1.0) == CONSTANTS.charge_enable.MAX) ? getColor("txtGreenBg", colorMode) : getColor("txtRedBg", colorMode);
```
Sorry for going against the design again, but:
- Same thing as before about making the text 'N/A' when there's no value for charge_enable
- After looking back at the description of the charge_enable signal, I changed up the wording and color logic a bit:
  - I think it will make more sense to use 'High' and 'Low' rather than 'Enable' and 'Disable'
  - Since the nominal value is 0/false, we should compare charge_enable to its nominal maximum value (0) when determining color (again, we'll want `==` for type coercion). I think charge_enable will always have a preferred value, but in the event it doesn't and the nominal range becomes 0-1 instead of only either 0 or 1, the color will be green for high and red for low, which fits the conventions of the other components we have.

Reply:
I decreased the width of the box holding the enable/disable text since we changed it to high/low, 3em seems to avoid making it look too big. If you want green for high and red for low, it's currently the opposite, unless I misunderstood your final bullet point: 
<img width="294" alt="image" src="https://github.com/badgerloop-software/chase-car-dashboard/assets/17338790/9e336a0c-6051-4762-af4f-2e1c570db8c0">
<img width="297" alt="image" src="https://github.com/badgerloop-software/chase-car-dashboard/assets/17338790/8b507ae4-2e3a-497b-b9ea-a260a904bf38">

Reply:
The images you have look correct (low->green and high->red)